src/LinAlg/hiopMatrixComplexDense.cpp

```diff
@@ -190,25 +189,27 @@ namespace hiop
       if(maxRows>m_local_) maxRows=m_local_;
       if(maxCols>n_local_) maxCols=n_local_;
       
-      if(f==NULL) f=stdout;
-      
       if(msg) {
-	fprintf(f, "%s (local_dims=[%d,%d])\n", msg, m_local_,n_local_);
+        fprintf(f, "%s (local_dims=[%d,%d])\n", msg, m_local_,n_local_);
       } else { 
-	fprintf(f, "hiopMatrixComplexDense::printing max=[%d,%d] (local_dims=[%d,%d], on rank=%d)\n", 
-		maxRows, maxCols, m_local_,n_local_,myrank_);
+        fprintf(f, "hiopMatrixComplexDense::printing max=[%d,%d] (local_dims=[%d,%d], on rank=%d)\n", 
```

Comment:
codestyle need to be enforced: one liners maybe, spaces missing

Reply:
it is too long to fit in one line. Hence I I just added the missing spaces.

Reply:
see https://github.com/LLNL/hiop/blob/develop/src/Optimization/hiopNlpFormulation.cpp#L1203-L1209 for example. I think we somehow follow the same coding style.